nucypher/blockchain/eth/deployers.py

```diff
@@ -168,6 +170,9 @@ def deploy(self, gas_limit: int = None) -> dict:
         contract, deployment_receipt = self.blockchain.deploy_contract(self.contract_name,
                                                                        self.__economics.erc20_total_supply,
                                                                        gas_limit=gas_limit)
+        if progress:
+            progress.update(1)
+            time.sleep(3)
```

Comment:
Why sleep here?

Reply:
Whoops! That was from testing a slowing deployment - I'll remove it

Reply:
^ That was outdated - try refreshing the file.